doc/man/8/cephadm.rst

```diff
@@ -360,6 +371,27 @@ Pull the ceph image::
 
     cephadm pull
 
+registry-login
+--------------
+
+Give cephadm login information for an authenticated registry (url, username and password).
+Cephadm will attempt to login to that registry and store the login information in the cluster's
+configuration database::
+
+      cephadm registry-login --registry-url [REGISTRY_URL] --registry-username [USERNAME]
```

Comment:
`podman` also have a `--tls-verify` option that allows login into registries without SSL, but this option is not available on `docker`.

@sebastian-philipp what it the approach used in `cephadm`? Only support options that can be used by both `podman` and `docker`? Or support options that can only be used by one of them, and throw an error if you use that option on the wrong engine?

Reply:
I would think a better approach might would be to configure podman/docker with a self-signed cert and/or as an insecure registery ...